packages/react-mutation-mapper/src/component/column/ClinVar.tsx

```diff
@@ -17,10 +20,13 @@ export function download(myVariantInfo?: MyVariantInfo): string {
     return value ? value.toString() : '';
 }
 
-export function sortValue(myVariantInfo?: MyVariantInfo): number | null {
-    const id = getClinVarId(myVariantInfo);
+export function sortValue(myVariantInfo?: MyVariantInfo): string | null {
```

Comment:
seems pretty heavy for a sort.  will this be used on a large collection?

Reply:
This one is used to sort the column in the table. So, the collection size is the number of mutations. For a table with ~5000 mutations sorting is pretty fast. If we happen to have issues at some point, we can either try to improve this function or disable sorting for this column. Since this column is hidden by default not sure if it is worth to optimize sort at this point.

Reply:
Does it sort by alphabetical order?

Reply:
yes, but it might be better to sort like `Oncogenic > Likely Oncogenic > ...`